src/components/FeatureLists/FeatureLists.tsx

```diff
@@ -2,23 +2,19 @@ import React from 'react';
 import { Grid, Heading } from '@aws-amplify/ui-react';
 
 interface FeatureListsProps {
-  children: React.ReactNode;
-  platform: string;
+  children?: React.ReactNode;
+  level?: 1 | 2 | 3 | 4 | 5 | 6;
```

Comment:
Since we can specify the heading level of the individual lists, let's make it so we can specify for the parent also. Defaulted to 2